LambdaEngine/Include/Rendering/Core/API/GraphicsDevice.h

```diff
@@ -99,13 +108,13 @@ namespace LambdaEngine
 
 		virtual DescriptorSet*	CreateDescriptorSet(const String& debugName, const PipelineLayout* pPipelineLayout, uint32 descriptorLayoutIndex, DescriptorHeap* pDescriptorHeap) const = 0;
 
-		virtual RenderPass*		CreateRenderPass(const RenderPassDesc* pDesc)	 const = 0;
+		virtual RenderPass*		CreateRenderPass(const RenderPassDesc* pDesc) const = 0;
```

Comment:
The class 'GraphicsDevice' which has virtual members does not have a virtual destructor.